### ISSUE TITLE
Performance issue

### ISSUE THREAD

**Issue by tonysy** [junfu1115/DANet#14]
I clone your code and run the training script.
But, I only get the performance at 73.5% mIoU on val set(CityScapes)
Is there any training tricks?

**Comment by junfu1115** [comment]
We run our code for several times with the training script, the performances are above 79.5% on val set. Is your training code exactly the same as ours?

**Comment by tonysy** [comment]
Yes, I clone your code and run your script. I use 4xP40 with PyTorch 0.4.1(install by pip). Could you share your hardware information and system version, and other information in detail? If you could provide a docker file, it would be great.

**Comment by junfu1115** [comment]
We also use P40,  CUDA 8.0 and Anaconda 3. We adopt PyTorch commit fd25a2a, u can ref https://github.com/junfu1115/DANet/issues/17#issuecomment-425388967.

**Comment by zhijiew** [comment]
@tonysy Can you get a better performance now? My result is also 73%-74%.

**Comment by tonysy** [comment]
@zhijiew  The results in training is not the real mIoU, you need to run the test.py to report the real mIoU results. I could get 77~78% on validation set when training result is 73%~74%.

@junfu1115 Does it make sense?

**Comment by zhijiew** [comment]
@tonysy Thank you! My result is reasonable now :)
